h/templates/includes/annotation_card.html.jinja2

```diff
@@ -62,7 +62,14 @@
   <footer class="annotation-card__footer">
     {% if presented_annotation.incontext_link %}
       <a href="{{ presented_annotation.incontext_link }}" target="_blank" title="{% trans %}Visit annotation in context{% endtrans %}">
-        {{ svg_icon('up-right-arrow', 'annotation-card__incontext-link') }}
+        {{ svg_icon('up-right-arrow', 'annotation-card__footer-link annotation-card__incontext-link') }}
+      </a>
+      <a href="#" share-widget-config='{
+        "url": "{{presented_annotation.incontext_link}}",
+        "private": {%if presented_annotation.annotation.shared and not presented_annotation.group %}true{%else%}false{%endif%},
```

Comment:
This conditional is the wrong way round - `private` should be true if the annotation is _not_shared.